src/test/suite/fixtures/recorded/languages/rust/chuckArgDrum.yml

```diff
@@ -0,0 +1,34 @@
+languageId: rust
+command:
+  spokenForm: chuck arg drum
+  version: 2
+  targets:
+    - type: primitive
+      mark: {type: decoratedSymbol, symbolColor: default, character: d}
+      modifiers:
+        - type: containingScope
+          scopeType: {type: argumentOrParameter}
+  usePrePhraseSnapshot: true
+  action: {name: remove}
+initialState:
+  documentContents: |
+    #[derive(Debug)]
+    sturct S {}
+  selections:
+    - anchor: {line: 2, character: 0}
+      active: {line: 2, character: 0}
+  marks:
+    default.d:
+      start: {line: 0, character: 9}
+      end: {line: 0, character: 14}
+finalState:
+  documentContents: |
+    #[derive()]
```

Comment:
Nice. Can you also add some tests with `cfg` and `cfg_attr` attributes? For example:
```rust
#[cfg(test)]
#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
struct Test;
```
I'd expect to be able to refer to `name`/`value` in `feature = "serde"`.